prompting/organic/organic_scoring_prompting.py

```diff
@@ -176,7 +173,7 @@ async def stream_miner_chunks(uid: int, chunks: AsyncGenerator):
             return_exceptions=True,
         )
 
-    async def _reuse_organic_response(self, sample: OrganicEntry) -> dict[int, SynapseStreamResult]:
+    async def _reuse_organic_response(self, sample: dict[str, Any]) -> dict[int, SynapseStreamResult]:
```

Comment:
nit: if we can avoid using the type `Any`, the better.

Reply:
Added dataclasses and more precise type hints. Changing `sample` to dataclass is some bigger change, which requires modifying organic-scoring repo